src/lightning/pytorch/loggers/neptune.py

```diff
@@ -43,12 +43,14 @@
     import neptune
     from neptune import Run
     from neptune.handler import Handler
+    from neptune.types import File
     from neptune.utils import stringify_unsupported
 elif _NEPTUNE_CLIENT_AVAILABLE:
     # <1.0 package structure
     import neptune.new as neptune
     from neptune.new import Run
     from neptune.new.handler import Handler
+    from neptune.new.types import File
```

Comment:
File = None needs to be included in the "else" below, so it can be mocked in the tests (see test failure)